torch/csrc/distributed/c10d/TraceUtils.h

```diff
@@ -256,4 +258,260 @@ inline std::string retrieveDesyncReport(
   return report;
 }
 
+/* Trace Utils Related to Flight Recorder */
```

Comment:
maybe also comment that this is only used by PGNCCL (could be generalized in an ideal world but wasn't done that way, so isn't expected to be fully general at the moment)

Reply:
Thanks, done!